cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -38,6 +38,14 @@ fn int_ty_to_is64(ty: Type) -> bool {
     }
 }
 
+fn flt_ty_to_is64(ty: Type) -> bool {
```

Comment:
I'd prefer you called this `ty_is_flt64` and rename its integer counterpart to `ty_is_int64`.